package.json

```diff
@@ -30,18 +30,17 @@
     ]
   },
   "devDependencies": {
-    "@backstage/cli": "^0.22.9",
-    "@changesets/cli": "^2.26.1",
-    "@spotify/prettier-config": "^12.0.0",
-    "cross-var": "^1.1.0",
-    "eslint-plugin-import": "^2.27.5",
-    "husky": "^8.0.3",
-    "prettier": "^2.3.2",
-    "turbo": "^1.9.3",
-    "typescript": "^5.0.4"
+    "@backstage/cli": "0.22.12",
+    "@changesets/cli": "2.26.2",
+    "@spotify/prettier-config": "15.0.0",
+    "cross-var": "1.1.0",
```

Comment:
Concurrently dependency is missing

Reply:
Concurrently is not needed because of `turbo`.